Update upload streaming for release. (#6246)
GoogleChrome/web.dev: src/site/content/en/blog/fetch-upload-streaming/index.md

authors:
  - jakearchibald
description: >
  Chrome now supports upload streaming as of version 95, which means
  you can start a request before you have the whole body available.
date: 2020-07-22
updated: 2021-09-22
hero: image/admin/9U7u4C7WCGbrdHm3181W.jpg
alt: A canoe pointed up stream.
tags:
  - blog
  - chrome95
  - network
  - service-worker
feedback:
  - api
---

> The origin trial is and request streams are available starting Chrome 95. This
> means you can start a request before you have the whole body available.

Reviewer: ```suggestion
From Chrome 95, you can start a request before you have the whole body available by using the streams API.
```
